docs/change_log/release-3.4.md

```diff
@@ -0,0 +1,42 @@
+title: Release Notes for v3.4
+
+# Python-Markdown 3.4 Release Notes
+
+Python-Markdown version 3.4 supports Python versions 3.6, 3.7, 3.8, 3.9 and PyPy3.
+
+## Backwards-incompatible changes
+
+### The `table` extension now uses a `style` attribute instead of `align` attribute for alignment.
+
+The [HTML4 spec][spec4] specifically
+deprecates the use of the `align` attribute and it does not appear at all in the 
+[HTML5 spec][spec5]. Therefore, by default, the [table] extension will now use the `style`
+attribute (setting just the `text-align` property) in `td` and `th` blocks.
+
+The former behavior is available by setting the setting `use_align_attribute` configuration
+option to `True` when adding the extension.
+
+For example, to configure the old `align` behavior:
+
+```python
+from markdown.extensions.tables import TableExtension
+
+markdown.markdown(src, extensions=[TableExtensioon(use_align_attribute=True)])
```

Comment:
Typo: "TableExtensioon" -> "TableExtension"

Reply:
Thanks. Not sure how that happened, I thought I'd copied and pasted that from my other window.